### PR TITLE
Update maint list on changelog because im on it

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

from discord roles

### CODE REVIEW

File: html/templates/header.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
...
 			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, monster860, Partheo, Wejengin2<br></font>
+			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, cuackles, Djiq, Ling, Manatee, Molti, monster860, OrcaCora, Partheo, Wejengin2, ynot01, Anerisa, Halcyon, TheReddDragon<br></font>
 			<font size='2'><b>Spriters:</b> Alagoinha, Cuackles, Halcyon, Missatessatessy, Mqiib, OrcaCora, Tipy1802, Xoxeyos<br></font>
 			<font size='2'><b>Mappers:</b> Vaelophis, baiomu, Ezhan, MenacingManatee, N3D6, GravelyAnonymous, Wejengin2, Xoxeyos<br></font>
 			<font size='2'><b>Thanks to:</b> /tg/station, FTL13 devs, Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers, Invisty for the title image and Hippiestation for dissapointed.ogg and other assets.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on our discord.</font>
```
Comment (ReddicusDragon) [inline html/templates/header.html:43]:
```suggestion
			<font size='2'><b>Thanks to:</b> /tg/station, FTL13 devs, Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers, Invisty for the title image and Hippiestation for dissapointed.ogg and other assets.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on our Discord.</font>
```

File: html/templates/header.html [hunk 2]
```diff
@@ -37,7 +37,7 @@
 		<td valign='top'>
 			<font size='2'><b>Current Head Developers:</b> JamieD12<br></font>
 			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, monster860, Partheo, Wejengin2<br></font>
+			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, cuackles, Djiq, Ling, Manatee, Molti, monster860, OrcaCora, Partheo, Wejengin2, ynot01, Anerisa, Halcyon, TheReddDragon<br></font>
```
Comment (ReddicusDragon) [inline html/templates/header.html]:
```suggestion
			<font size='2'><b>Maintainers:</b> adamsogm, AshCorr, baiomu, Bibby, Cuackles, Djiq, Ling, Manatee, Molti, monster860, OrcaCora, Partheo, Wejengin2, ynot01, Anerisa, Halcyon, TheReddDragon<br></font>
```
Capitalization correction

Comment (ynot01) [reply html/templates/header.html]: cuackles has his name set to lowercase on discord

File: html/templates/header.html (hunk 1 above)
Comment (ynot01) [reply html/templates/header.html:43]: out of scope

File: html/templates/header.html (hunk 2 above)
Comment (ReddicusDragon) [reply html/templates/header.html]: > cuackles has his name set to lowercase on discord

I have asked him how he prefers it and he says capital

**Review by Moltijoe — APPROVED** [review]
Very important

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**